README.md

```diff
@@ -21,16 +21,15 @@ We have created some deployments to get you started in just a few seconds. (But
 
 ### OpenShift:
 ```bash
-oc create -fhttps://github.com/tnozicka/openshift-acme/raw/master/deploy/{deploymentconfig-letsencrypt-staging,service}.yaml
+oc create -fhttps://github.com/tnozicka/openshift-acme/raw/master/deploy/{clusterrole,deploymentconfig-letsencrypt-staging,service}.yaml
 ```
 #### Privileges
 Because the controller needs to watch for events across namespaces and write certificate objects to them, depending on the settings, it needs elevated privileges. One way to allow necessary privileges is shown bellow, but you might set up more specific policy if you like.
 ```bash
-oadm policy add-cluster-role-to-user cluster-admin system:serviceaccount:acme:default
+oc adm policy add-cluster-role-to-user acme-controller system:serviceaccount:acme:default
 ```
```

Comment:
Since we are restricting what the controller can do and a lot of the functionality isn't covered by tests yet I would prefer to leave a fallback here:

If, for some reason, this doesn't work for you, please file an issue and you can fallback to elevated privileges by using the default `cluster-admin` role:
`oc adm policy add-cluster-role-to-user cluster-admin system:serviceaccount:acme:default`